### ISSUE TITLE
Autocomplete : Popover (dropdown) does not stay underneath

### ISSUE THREAD

**Issue by AkselsLedins** [marmelab/react-admin#2370]
Hello !

**What you were expecting:**
That the dropdown stays underneath and won't cover the input.

**What happened instead:**
The dropdown is covering the input.

**Steps to reproduce:**
Works in demo app:
https://marmelab.com/react-admin-demo/#/reviews?filter=%7B%7D&order=DESC&page=1&perPage=25&sort=date

* Add filter -> Customer
* Click on customer reference input (filter)
* Scroll down a little

![popover-bug](https://user-images.githubusercontent.com/18367406/46276767-69a94280-c561-11e8-8583-6449819f5e93.gif)

you can click outside and click again the input, the dropdown will still be on top on the input, preventing you from typing anything.

It's really annoying :disappointed:

**Comment by FACOLOMBANI** [comment]
I am having the same problem when using AutocompleteArrayInput and AutocompleteInput below a custom component that loads after both inputs.
![autocompleteerror](https://user-images.githubusercontent.com/11721821/46660570-98559780-cbb7-11e8-973b-729556e54a57.gif)

**Comment by FACOLOMBANI** [comment]
I fixed my issue by adding a the prop 'disablePortal={true}' to the Popper component in the renderSuggenstionsContainer.

**Comment by afilp** [comment]
@FACOLOMBANI Whas that change inside the `react-admin` code? Or in your project's code? Thanks.

**Comment by FACOLOMBANI** [comment]
I did this inside react-admin code. I forked the component for customization because i needed to group choices, render groups and secondary information, so that was the  fix to the Popper in Both AutocompleteArrayInput and AutocometeInput. In this page you will find the description for disablePortal prop https://material-ui.com/api/popper/. Hope it helps.

**Comment by FACOLOMBANI** [comment]
@aflip i looked at my code again and saw i also added a modifier in the Popper component, look at the modifiers prop below 

Here is the description https://popper.js.org/popper-documentation.html#modifiers..keepTogether

```
<Popper
                className={className}
                open
                anchorEl={this.inputEl}
                disablePortal={true}
                placement="bottom-start"
                modifiers={{
                    keepTogether: {
                        enabled: true
                    }
                }}
            >
                <Paper square className={classes.suggestionsPaper} {...containerProps}>
                    {children}
                </Paper>
            </Popper>
```

I could create a PR for this.

**Comment by afilp** [comment]
Wait, I just created one, I am pusing now! :-)

**Comment by FACOLOMBANI** [comment]
@afilp The first work around for me was the one you just sent, but i needed the flip enabled to true, and i didn't needed the preventOverflow, and using keepTogether worked well with the flip.

**Comment by afilp** [comment]
@FACOLOMBANI Thanks! Not sure why I did not see it working in my case. @djhi replied to the PR and he will make a better, more generic PR.

**Comment by fzaninotto** [comment]
Fixed by #2678
